pygmt/src/plot3d.py

```diff
@@ -153,6 +153,12 @@ def plot3d(
         ``pen``.
     {c}
     {f}
+    columns : str or 1d array
+        Choose which columns are x, y, color, and size, respectively if
+        input is provided via *data*. E.g. ``columns = [0, 1]`` or
+        ``columns = '0,1'`` if the *x* values are stored in the first
```

Comment:
```suggestion
        ``columns='0,1'`` if the x values are stored in the first
```
Most other examples in the API docs do not use spaces after the parameters
The x and y values are not italicized in ``x/y/z`` or ``data``